docs/dev/api-tests.rst

```diff
@@ -3,6 +3,24 @@ API Contract Tests
 These tests are designed to look for changes to the recipe server API that are
 not expected.
 
+Before running these tests you will need the following installed:
+
+.. Python 3.6.0 or greater
+.. _virtualenv: https://virtualenv.pypa.io/en/latest/virtualenv installed
```

Comment:
This isn't the syntax for lists in markdown. I think the right syntax would be

```rst
* Python 3.6.0 or greater
* virtualenv_

.. _virtualenv: https://virtualenv.pypa.io/en/latest/virtualenv
```

That would create a list of two elements, the second of which is a link.